src/main/java/com/sda/groupa/shippingcostcalculator/login/config/SecurityConfig.java

```diff
@@ -50,14 +57,18 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
     }
 
     @Bean
-    AuthenticationSuccessHandler successHandler() {
-        return new SavedRequestAwareAuthenticationSuccessHandler();
+    public AuthenticationSuccessHandler myAuthenticationSuccessHandler(){
```

Comment:
Zmieńmy nazwę tej metody na `userLoginSuccessHandler`, żeby wskazywała na to co dostarcza.

Reply:
Właśnie nie widzę tego u siebie, tak jak by ten plik nie został zaktualizowany